[2023-11-05T15:42:28.153+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: customers.create_customers_table manual__2023-11-05T15:42:26.061449+00:00 [queued]>
[2023-11-05T15:42:28.168+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: customers.create_customers_table manual__2023-11-05T15:42:26.061449+00:00 [queued]>
[2023-11-05T15:42:28.169+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T15:42:28.170+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-11-05T15:42:28.172+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T15:42:28.203+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_customers_table> on 2023-11-05 15:42:26.061449+00:00
[2023-11-05T15:42:28.212+0000] {standard_task_runner.py:55} INFO - Started process 15803 to run task
[2023-11-05T15:42:28.216+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'customers', 'create_customers_table', 'manual__2023-11-05T15:42:26.061449+00:00', '--job-id', '427', '--raw', '--subdir', 'DAGS_FOLDER/customers.py', '--cfg-path', '/tmp/tmp3fjigmc3']
[2023-11-05T15:42:28.219+0000] {standard_task_runner.py:83} INFO - Job 427: Subtask create_customers_table
[2023-11-05T15:42:28.289+0000] {task_command.py:376} INFO - Running <TaskInstance: customers.create_customers_table manual__2023-11-05T15:42:26.061449+00:00 [running]> on host 9d85c6014d6e
[2023-11-05T15:42:28.534+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=customers
AIRFLOW_CTX_TASK_ID=create_customers_table
AIRFLOW_CTX_EXECUTION_DATE=2023-11-05T15:42:26.061449+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-05T15:42:26.061449+00:00
[2023-11-05T15:42:28.545+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-11-05T15:42:28.550+0000] {sql.py:315} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS customers (
            Index INT NOT NULL primary key,
            Customer_Id varchar(100),
            First_Name varchar(100),
            Last_Name varchar(100),
            Company varchar(100),
            City varchar(100),
            Country varchar(100),
            Phone_1 varchar(100),
            Phone_2 varchar(100),
            Email varchar(100),
            Subscription Date,
            Website varchar(100)
        );
    , parameters: None
[2023-11-05T15:42:28.551+0000] {postgres.py:96} INFO - NOTICE:  relation "customers" already exists, skipping

[2023-11-05T15:42:28.562+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=customers, task_id=create_customers_table, execution_date=20231105T154226, start_date=20231105T154228, end_date=20231105T154228
[2023-11-05T15:42:28.588+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-05T15:42:28.623+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
